app/presenters/setting_presenter.rb

```diff
@@ -50,6 +50,10 @@ def readonly?
     SETTINGS.key?(name.to_sym)
   end
 
+  def value
+    readonly? ? SETTINGS[name.to_sym] : super
```

Comment:
This does two lookups in `SETTINGS`. Can't we use `fetch()`?
```suggestion
    SETTINGS.fetch(name.to_sym) { super }
```

Reply:
It does two only for the global settings, so it boils down to whether is `key?` more efficient then `fetch`, but I guess it's insignificant, so using yours ;)